system/ee/ExpressionEngine/Controller/Settings/Captcha.php

```diff
@@ -51,6 +51,34 @@ public function index()
                         'captcha_rand' => array('type' => 'yes_no')
                     )
                 ),
+                array(
+                    'title' => 'use_recaptcha',
+                    'desc' => 'use_recaptcha_desc',
+                    'fields' => array(
+                        'use_recaptcha' => array('type' => 'yes_no')
+                    )
+                ),
+                array(
+                    'title' => 'recaptcha_site_key',
+                    'desc' => 'recaptcha_site_key_desc',
+                    'fields' => array(
+                        'recaptcha_site_key' => array('type' => 'text')
+                    )
+                ),
+                array(
```

Comment:
Not sure what the EE team thinks about this, but IMO arrays should start using the short syntax.

Reply:
I discussed this with Geoff and I think it's fine to leave this in the old `array()` syntax for now because that's how the rest of the file is and it would be odd to start mixing styles unless we refactor the entire file.